examples/eks_cluster_webshop/main.tf

```diff
@@ -64,9 +64,6 @@ module "castai-eks-cluster" {
   aws_instance_profile_arn      = module.castai-eks-role-iam.instance_profile_arn
 
   subnets         = var.subnets
-  # As for now (05.05.2022) you need to fill security groups after cluster onboarding with values from plan.  
-  security_groups =  []
```

Comment:
Maybe keep example as `override_security_groups = []`

Reply:
This will remove whole SGs for the cluster.  Default is null, empty array will set SGs to empty array